src/EditorFeatures/Core.Wpf/InlineRename/AbstractEditorInlineRenameService.SymbolRenameInfo.cs

```diff
@@ -53,6 +54,7 @@ private partial class SymbolInlineRenameInfo : IInlineRenameInfo
                 TextSpan triggerSpan,
                 SymbolAndProjectId renameSymbolAndProjectId,
                 bool forceRenameOverloads,
+                bool isLanguageCaseSensitive,
```

Comment:
you have document, so you can use this http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs,14

rather than having it as field.